README.md

```diff
@@ -10,6 +10,16 @@ Debezium is a distributed platform that turns your existing databases into event
 
 Debezium is open source under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
 
+# Purpose of the Debezium UI
+The Debezium UI provides standalone web application, which connects to Kafka Connect via its REST API. See the [Debezium UI docs](https://debezium.io/documentation/reference/operations/debezium-ui.html) for more information
+
+# Debezium environment vars
+The table shows the ENV vars for the [Debezium UI container image](https://hub.docker.com/r/debezium/debezium-ui) and it's related `application.properties` parameter name when running the java application locally.
+
+| ENV vars        | application.properties vars  | Default value             | Description |
+| :-------------- |:-----------------------------|:--------------------------|:------------|
+| DEPLOYMENT_MODE | deployment.mode              | default                   | Parameter to enable/disable connector properties validation mode in the Debezium UI, to disable the validation change value to `validation.disabled` |
+| KAFKA_CONNECT_URI|kafka.connect.uri            | http://connect:8083       | Provide existing Kafka connection instance url. `KAFKA_CONNECT_URI` supplies the comma-separated list of available URI(s) |
```

Comment:
@uidoyen that looks already quite good. Maybe you can take a look into the Debezium connectors documentation and see how we document the connector configuration in such tables there. 
We normally list the available options with a brief description. Maybe we can do this here as well?

Reply:
@rk3rn3r updated the vars description accordingly.